gwctl/pkg/printer/backends.go

```diff
@@ -49,59 +49,36 @@ func (bp *BackendsPrinter) Print(resourceModel *resourcediscovery.ResourceModel)
 		os.Exit(1)
 	}
 
-	backendNodes := make([]*resourcediscovery.BackendNode, 0, len(resourceModel.Backends))
-	for _, backendNode := range resourceModel.Backends {
-		backendNodes = append(backendNodes, backendNode)
-	}
+	backends := common.MapToValues(resourceModel.Backends)
```

Comment:
Perhaps we can start using the same `maps.Values()` here as well (and eventually get rid of common.MapToValues in the other places as well -- you need not make all the other changes here)

Reply:
done!